content/docs/quickstart/sqlalchemy.md

```diff
@@ -2,13 +2,23 @@
 title: Run a SQL Alchemy App
 ---
 
+### Introduction
+
 SQLAlchemy is among the most popular ORMs in the Python universe. Because Neon is fully compatible with vanilla PostgreSQL, you only need to fill in the correct connection details.
 
-Prerequisites:
+## Prerequisites
+
+To complete this section, you will need to already have the following:
+
+- Created a Project on Neon Cloud service
```

Comment:
Nit: should we create a separate section on creating projects? It's more or less the same text in multiple guides.

Reply:
@yeputons 

There is a section for setting up a project, please look at the sitemap, https://www.figma.com/file/TPDx7Xb0szv4NIyteVmVrL/Neon---Site-map-May-2022-(Copy)?node-id=0%3A1.

This is the text cleaned up but already posted from the original doc owner in multiple guides.